multicompany_property_base/models/res_partner.py

```diff
@@ -32,6 +32,11 @@ def _compute_properties(self):
                 values.append(val.id)
             record.property_ids = values
 
+    def _commercial_sync_to_children(self):
+        return super(
+            Partner, self.with_context(commercial_sync_to_children=True)
+        )._commercial_sync_to_children()
+
```

Comment:
this context should be put in the codebase where it is needed as here it is not needed

Reply:
just changed it!